### PR TITLE
base1: simplify the Content-Length test

### PR DESCRIPTION
**Pull request by jelly**
Get rid of the bash wrapping and invoke stat directly, make the request synchronous.

### CODE REVIEW

File: pkg/base1/test-external.js [hunk 1]
```diff
@@ -44,18 +44,15 @@ QUnit.test("external fsread1", async assert => {
...
-            assert.equal(req.getResponseHeader("Content-Type"), "application/octet-stream", "default type");
-            assert.equal(req.getResponseHeader("Content-Disposition"), 'attachment; filename="foo"', "default type");
-            assert.equal(req.getResponseHeader("Content-Length"), parseInt(filesize), "expected file size");
-            done();
-        }
-    };
+    req.open("GET", "/cockpit/channel/" + cockpit.transport.csrf_token + '?' + query, false);
     req.send();
+
+    assert.equal(req.status, 200, "got right status");
```
Comment (martinpitt) [inline pkg/base1/test-external.js:50]: Could you please split out that change into a separate commit? At first sight I don't see how that would work? That's an XHR request, which is asynchronous. How can this be done immediately? Does `.send()` block until the reply comes back? That'd be *very* un-JS-y.

Comment (jelly) [reply pkg/base1/test-external.js]: done

**Review by allisonkarlitskaya — APPROVED** [review]
Looks good, thanks.  Please trigger the mandatory test scenario.
